First/src/tutorials/MultipleArray.java

```diff
@@ -0,0 +1,35 @@
+package tutorials;
+
+public class MultipleArray {
+
+	public static void main(String[] args) 
+	{
+		int a [][]= new int[2][2]; //{{0,9},{5,1,6},{2,2,7}};
+		a[0][0]=1;
+		a[0][1]=2;
+		a[1][0]=3;
+		a[1][1]=4;
+
+		for (int i=0;i<2;i++)
```

Comment:
Please change variable names here. Give proper names. Please look other places also and change there even if comment is not there.

Reply:
Done. Updated code !

Reply:
I forgot to delete this file from git. In the last week I have added another file for Linkedlist named as CharacterLinkedList. Please review this file.